misk/src/test/kotlin/misk/web/WebDispatchTest.kt

```diff
@@ -0,0 +1,134 @@
+package misk.web
+
+import com.google.common.truth.Truth.assertThat
+import com.google.common.util.concurrent.ServiceManager
+import com.google.inject.Guice
+import com.google.inject.Provides
+import com.squareup.moshi.Moshi
+import misk.MiskModule
+import misk.inject.KAbstractModule
+import misk.inject.uninject
+import misk.testing.PortPicker
+import misk.web.actions.WebAction
+import okhttp3.HttpUrl
+import okhttp3.MediaType
+import okhttp3.OkHttpClient
+import okhttp3.Request
+import okhttp3.RequestBody
+import org.junit.After
+import org.junit.Before
+import org.junit.Test
+import java.util.concurrent.TimeUnit
+import javax.inject.Inject
+import javax.inject.Singleton
+
+internal class WebDispatchTest {
+    data class HelloBye(val message: String)
+
+    val jsonMediaType = MediaType.parse("application/json")
+
+    @Inject lateinit var serviceManager: ServiceManager
+    @Inject lateinit var webConfig: WebConfig
+    @Inject lateinit var moshi: Moshi
+
+    @Before
+    fun before() {
+        uninject(this)
```

Comment:
this is an `@After` thing, isn’t it?

Reply:
Yes. Also it goes away entirely in the content based routing branch.